HLTrigger/Configuration/python/HLT_75e33_cff.py

```diff
@@ -404,7 +405,7 @@
     fragment.HLT_DoubleMediumChargedIsoPFTauHPS40_eta2p1,
     fragment.HLT_DoubleMediumDeepTauPFTauHPS35_eta2p1,
     fragment.HLT_IsoMu20_eta2p1_LooseDeepTauPFTauHPS27_eta2p1_CrossL1,
-
+    fragment.HLT_Ele30_WPTight_L1Seeded_LooseDeepTauPFTauHPS30_eta2p1_CrossL1,
```

Comment:
please add this trigger also in the timing variant of the menu at `HLTrigger/Configuration/python/HLT_75e33_timing_cff.py`

Reply:
I have added this E-Tau path to the HLT_75e33_timing_cff.py file